knimin/handlers/pm_shotgun_pool.py

```diff
@@ -0,0 +1,76 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) 2014--, The LabAdmin Development Team.
+#
+# Distributed under the terms of the BSD 3-clause License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# -----------------------------------------------------------------------------
+
+import datetime
+
+from tornado.web import authenticated
+
+from knimin import db
+from knimin.handlers.base import BaseHandler
+from knimin.handlers.access_decorators import set_access
+from knimin.lib.parse import parse_qpcr_object
+from knimin.lib.shotgun import (compute_qpcr_concentration,
+                                compute_shotgun_pooling_values_qpcr)
+
+
+@set_access(['Admin'])
+class PMShotgunPool(BaseHandler):
+    @authenticated
+    def get(self):
+        plate_id = int(self.get_argument('plate_id'))
+
+        res = db.read_normalized_shotgun_plate(plate_id)
+        machines = db.get_property_options('qpcr')
+
+        # echo is the name of the plate
+        self.render('pm_shotgun_pool.html', name=res['echo'],
```

Comment:
BTW, failures are unrelated to the changes here - they're a result of the recent merge on AG-web